src/daemon/ca/server.rs

```diff
@@ -217,41 +228,50 @@ impl CaServer {
 
     /// Republish the embedded TA and CAs if needed, i.e. if they are close
     /// to their next update time.
-    pub async fn republish_all(&self, actor: &Actor) -> KrillResult<()> {
-        for ca in self.ca_list(actor)?.cas() {
-            if let Err(e) = self.republish(ca.handle(), actor).await {
-                error!("ServerError publishing: {}, ServerError: {}", ca.handle(), e)
-            }
+    pub async fn republish_all(&self) -> KrillResult<Vec<Handle>> {
+        self.ca_objects_store.reissue_all()
```

Comment:
Previously, and I'm not sure it was correct, the code ONLY republished (reissued? why the different name?) for CAs that the caller had CA_READ permission on (which was handled by `ca_list(actor)`). That check is gone in the new implementation. Should there be some sort of check like this?